gnucash/import-export/qif-imp/assistant-qif-import.c

```diff
@@ -1796,11 +1800,12 @@ gnc_ui_qif_import_load_progress_start_cb (GtkButton * button,
 
     /* This step will fill 70% of the bar. */
     gnc_progress_dialog_push (wind->load_progress, 0.7);
-    load_return = scm_call_4 (qif_file_load,
+    load_return = scm_call_5 (qif_file_load,
                               SCM_CAR(imported_files),
                               scm_from_locale_string (path_to_load ? path_to_load : ""),
```

Comment:
Not yours but probably wrong: path_to_load is coming from a Gtk widget and will be UTF8, not the locale encoding.